lib/geo_pattern/errors.rb

```diff
@@ -0,0 +1,7 @@
+module GeoPattern
+  # user errors
+  class UserError < StandardError; end
```

Comment:
I think using this kind of pattern for exceptions is a question of style. If you do not like it, I can remove the "intermediate" error.